src/core/include/units/bits/pow.h

```diff
@@ -40,7 +41,9 @@ constexpr std::intmax_t ipow10(std::intmax_t exp)
 }
 
 template<typename Rep>
-constexpr Rep fpow10(std::intmax_t exp)
+constexpr Rep fpow10(std::intmax_t exp) noexcept(
+  std::is_nothrow_constructible_v<Rep, double>&& std::is_nothrow_move_assignable_v<Rep>&& noexcept(
+    std::declval<Rep>() / std::declval<Rep>()) && noexcept(std::declval<Rep>() * std::declval<Rep>()))
```

Comment:
Why do we need that? The aggreed LEWG policy in the ISO C++ Committee is that we should not use conditional `noexcept` for functions other than special member functions and `swap()`.

Reply:
Did not see that comment.
We have template, Rep could be what ever it wants, right? Including my own type which may throw exceptions just as I want.

Reply:
Yes, and that is why it should be left as potentially throwing rather than trying hard to specify when it is not. See the LEWG policy I provided in another comment.